dds_web/api/project.py

```diff
@@ -298,3 +300,70 @@ def put(self):
                 break
 
         return flask.jsonify({"updated": updated, "error": error, "tries": current_try})
+
+
+class CreateProject(flask_restful.Resource):
+    @auth.login_required
+    def post(self):
+        """Create a new project"""
+        p_info = flask.request.json
+        cur_user = auth.current_user()
+        # Check if the user actually exists
+        if not cur_user or cur_user.role != "researcher":  # role?
```

Comment:
This part will be handled by the `@auth.login_required` part so you can remove this

Reply:
👍